### ISSUE TITLE
"Unable to generate source map" error for some packages, using 5.1.0

### ISSUE THREAD

**Issue by brianlukoff** [zodern/minify-js-sourcemaps#34]
I'm getting this error for a few packages when running in development mode (for example, `risul:moment-timezone@0.5.7`):

```
Unable to generate source map for packages/[packagename]:
TypeError: Cannot read property 'index' of null
    at generateMap (packages/fastMinifier_plugin.js:443:40)
    at module.exports (packages/fastMinifier_plugin.js:455:12)
    at packages/fastMinifier_plugin.js:144:15
    at Array.forEach (<anonymous>)
    at MeteorBabelMinifier.<anonymous> (packages/fastMinifier_plugin.js:141:11)
    at MeteorBabelMinifier.processFilesForBundle (packages/fastMinifier_plugin.js:31:21)
    at __bottom_mark__ (/Users/brian/.meteor/packages/meteor-tool/.2.12.0.1wnk424.thrv++os.osx.arm64+web.browser+web.browser.legacy+web.cordova/mt-os.osx.arm64/tools/utils/tools/utils/parse-stack.ts:92:14)
    at /tools/isobuild/bundler.js:1427:11
    at Object.enterJob (/tools/utils/buildmessage.js:388:12)
    at ClientTarget.minifyJs (/tools/isobuild/bundler.js:1424:18)
    at /tools/isobuild/bundler.js:906:16
    at Object.enterJob (/tools/utils/buildmessage.js:388:12)
    at ClientTarget.make (/tools/isobuild/bundler.js:850:18)
    at /tools/isobuild/bundler.js:3281:14
    at /tools/isobuild/bundler.js:3441:25
    at Array.forEach (<anonymous>)
    at /tools/isobuild/bundler.js:3395:14
    at Object.capture (/tools/utils/buildmessage.js:283:5)
    at bundle (/tools/isobuild/bundler.js:3262:31)
    at /tools/isobuild/bundler.js:3205:32
    at Slot.withValue (/Users/brian/.meteor/packages/meteor-tool/.2.12.0.1wnk424.thrv++os.osx.arm64+web.browser+web.browser.legacy+web.cordova/mt-os.osx.arm64/dev_bundle/lib/node_modules/@wry/context/lib/context.esm.js:69:29)
    at Object.withCache (/Users/brian/.meteor/packages/meteor-tool/.2.12.0.1wnk424.thrv++os.osx.arm64+web.browser+web.browser.legacy+web.cordova/mt-os.osx.arm64/tools/fs/tools/fs/files.ts:1543:39)
    at Object.bundle (/tools/isobuild/bundler.js:3205:16)
    at /tools/runners/run-app.js:579:24
    at Function.run (/Users/brian/.meteor/packages/meteor-tool/.2.12.0.1wnk424.thrv++os.osx.arm64+web.browser+web.browser.legacy+web.cordova/mt-os.osx.arm64/tools/tool-env/tools/tool-env/profile.ts:289:14)
    at bundleApp (/tools/runners/run-app.js:578:34)
    at AppRunner._runOnce (/tools/runners/run-app.js:625:35)
    at AppRunner._fiber (/tools/runners/run-app.js:947:28)
    at /tools/runners/run-app.js:408:12
```

**Comment by alisnic** [comment]
getting this error as well for picker:

```
Unable to generate source map for packages/meteorhacks_picker.js:
TypeError: Cannot read property 'index' of null
    at generateMap (packages/fastMinifier_plugin.js:443:40)
    at module.exports (packages/fastMinifier_plugin.js:455:12)
    at MeteorBabelMinifier._minifyWithSwc (packages/fastMinifier_plugin.js:65:13)
    at MeteorBabelMinifier.minifyOneFile (packages/fastMinifier_plugin.js:122:19)
    at packages/zodern_caching-minifier.js:77:23
    at Function.Profile.time (packages/zodern_caching-minifier.js:34:5)
    at MeteorBabelMinifier.minifyFile (packages/zodern_caching-minifier.js:76:15)
    at packages/fastMinifier_plugin.js:184:27
    at Function.Profile.time (packages/fastMinifier_plugin.js:35:7)
    at packages/fastMinifier_plugin.js:183:17
    at Array.forEach (<anonymous>)
    at MeteorBabelMinifier.<anonymous> (packages/fastMinifier_plugin.js:164:9)
    at MeteorBabelMinifier.processFilesForBundle (packages/fastMinifier_plugin.js:31:21)
    at __bottom_mark__ (/home/circleci/.meteor/packages/meteor-tool/.2.12.0.fgp8sh.6bumo++os.linux.x86_64+web.browser+web.browser.legacy+web.cordova/mt-os.linux.x86_64/tools/utils/tools/utils/parse-stack.ts:92:14)
    at /tools/isobuild/bundler.js:1427:11
    at Object.enterJob (/tools/utils/buildmessage.js:388:12)
    at ClientTarget.minifyJs (/tools/isobuild/bundler.js:1424:18)
    at /tools/isobuild/bundler.js:906:16
    at Object.enterJob (/tools/utils/buildmessage.js:388:12)
    at ClientTarget.make (/tools/isobuild/bundler.js:850:18)
    at /tools/isobuild/bundler.js:3281:14
    at /tools/isobuild/bundler.js:3441:25
    at Array.forEach (<anonymous>)
    at /tools/isobuild/bundler.js:3395:14
    at Object.capture (/tools/utils/buildmessage.js:283:5)
    at bundle (/tools/isobuild/bundler.js:3262:31)
    at /tools/isobuild/bundler.js:3205:32
    at Slot.withValue (/home/circleci/.meteor/packages/meteor-tool/.2.12.0.fgp8sh.6bumo++os.linux.x86_64+web.browser+web.browser.legacy+web.cordova/mt-os.linux.x86_64/dev_bundle/lib/node_modules/@wry/context/lib/context.esm.js:69:29)
    at Object.withCache (/home/circleci/.meteor/packages/meteor-tool/.2.12.0.fgp8sh.6bumo++os.linux.x86_64+web.browser+web.browser.legacy+web.cordova/mt-os.linux.x86_64/tools/fs/tools/fs/files.ts:1543:39)
    at Object.bundle (/tools/isobuild/bundler.js:3205:16)
    at buildCommand (/tools/cli/commands.js:1159:30)
    at /tools/cli/commands.js:988:25
    at Function.run (/home/circleci/.meteor/packages/meteor-tool/.2.12.0.fgp8sh.6bumo++os.linux.x86_64+web.browser+web.browser.legacy+web.cordova/mt-os.linux.x86_64/tools/tool-env/tools/tool-env/profile.ts:289:14)
    at /tools/cli/commands.js:986:18
    at /home/circleci/.meteor/packages/meteor-tool/.2.12.0.fgp8sh.6bumo++os.linux.x86_64+web.browser+web.browser.legacy+web.cordova/mt-os.linux.x86_64/dev_bundle/lib/node_modules/meteor-promise/fiber_pool.js:43:40
Errors prevented bundling:                           
While minifying app code:
<anonymous>: Unexpected token u in JSON at position 0
at JSON.parse (<anonymous>)
at packages/fastMinifier_plugin.js:230:22
at Array.forEach (<anonymous>)
at packages/fastMinifier_plugin.js:225:21
at Function.Profile.time (packages/fastMinifier_plugin.js:35:7)
at MeteorBabelMinifier.<anonymous> (packages/fastMinifier_plugin.js:224:11)
at MeteorBabelMinifier.processFilesForBundle
(packages/fastMinifier_plugin.js:31:21)
```

**Comment by zodern** [comment]
This is fixed in version 5.1.1.
